cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,41 @@
-/// <reference types='cypress' />
-/// <reference types='../support' />
+import ArticlePageObject from '../support/pages/article.pageObject';
+
+const ArticlePage = new ArticlePageObject();
 
 describe('Article', () => {
-  before(() => {
+  let user;
+  let article;
 
+  before(() => {
+    cy.task('db:clear');
   });
 
   beforeEach(() => {
-    cy.task('db:clear');
+    cy.task('generateUser').then((generatedUser) => {
+      user = generatedUser;
+      cy.login(user);
+      cy.visit(`/#/editor`);
+    });
+    cy.task('generateArticle').then((data) => {
+      article = data;
+    });
   });
 
   it('should be created using New Article form', () => {
-
+    ArticlePage.createArticle(article);
   });
 
   it('should be edited using Edit button', () => {
-
+    ArticlePage.createArticle(article);
+    cy.task('generateArticle').then((newArticle) => {
+      ArticlePage.editArticle(article.title, newArticle);
+    });
   });
 
   it('should be deleted using Delete button', () => {
-
+    ArticlePage.createArticle(article);
+    ArticlePage.deleteArticleButton.click();
```

Comment:
Directly interacting with the delete button might not simulate a real user interaction accurately. Consider using a method from the ArticlePageObject to encapsulate this action, ensuring consistency and reusability.